bin/genesis

```diff
@@ -4190,18 +4321,18 @@ sub tablify {
 sub bosh_target_for {
 	my ($env) = @_;
 	if (defined($ENV{GENESIS_BOSH_ENVIRONMENT})) {
-		system_bosh("-e $ENV{GENESIS_BOSH_ENVIRONMENT} env") and
+		Genesis::ShellExec::bosh_interact('-e "$1" env', $ENV{GENESIS_BOSH_ENVIRONMENT}) and
 			return $ENV{GENESIS_BOSH_ENVIRONMENT};
 		die "No such host: $ENV{GENESIS_BOSH_ENVIRONMENT} from GENESIS_BOSH_ENVIRONMENT environment variable for the BOSH Director.\n";
```

Comment:
Lose the `-e` and set `BOSH_ENVIRONMENT` in the env hash.

Also, this should be a do_or_die, followed by a return of the env var